share/completions/dot.fish

```diff
@@ -0,0 +1,531 @@
+    'nodesep 0.25' \
+    'nojustify true false' \
+    'normalize false' \
+    'notranslate true false' \
+    nslimit \
+    nslimit1 \
+    'oneblock true false' \
+    'ordering in out' \
+    'orientation 0.0' \
+    'outputorder breadthfirst' \
+    'overlap true false' \
+    'overlap_scaling -4' \
+    'overlap_shrink true false' \
+    'pack true false' \
+    'packmode node' \
+    'pad 0.0555' \
+    page \
+    'pagedir BL' \
+    'quadtree normal none fase true false 2' \
+    quantum \
+    'rankdir 0.0' \
+    'ranksep 0.5' \
+    'ratio fill compress expand auto' \
+    'remincross true false' \
+    'repulsiveforce 1.0' \
+    'resolution 96.0' \
+    root \
+    'rotate 0' \
+    'rotation 0' \
+    scale \
+    'searchsize 30' \
+    'sep +4' \
+    'showboxes 0 1 2' \
+    size \
+    'smoothing none' \
+    'sortv 0' \
+    splines \
+    start \
+    'style filled invis' \
+    stylesheet \
+    target \
+    TBbalance \
+    tooltip \
+    truecolor \
+    URL \
+    viewport \
+    'voro_margin 0.05' \
+    xdotversion \
+    'sides 0'
+
+set -l edge_attributes class \
+    "color $(string join ' ' -- (__fish_graphviz__print_colors))" \
+    colorscheme \
+    comment \
+    "fillcolor $(string join ' ' -- (__fish_graphviz__print_colors))" \
+    "fontcolor $(string join ' ' -- (__fish_graphviz__print_colors))" \
+    'fontname Times-Roman' \
+    'fontsize 14.0' \
+    href \
+    id \
+    label \
+    layer \
+    'nojustify true false' \
+    'penwidth 1.0' \
+    pos \
+    shapefile \
+    'showboxes 0 1 2' \
+    'sides 0' \
+    'style filled invis' \
+    target \
+    tooltip \
+    URL \
+    xlabel \
+    xlp \
+    lp
+
+set -l node_attributes area \
+    class \
+    "color $(string join ' ' -- (__fish_graphviz__print_colors))" \
+    colorscheme \
+    comment \
+    'distortion 0.0' \
+    "fillcolor $(string join ' ' -- (__fish_graphviz__print_colors))" \
+    'fixedsize false true shape' \
+    "fontcolor $(string join ' ' -- (__fish_graphviz__print_colors))" \
+    'fontname Times-Roman' \
+    fontsize \
+    'gradientangle 0' \
+    group \
+    'height 0.5' \
+    href \
+    id \
+    image \
+    'imagepos tl tc tr ml mc mr bl bc br' \
+    'imagescale false true width height both' \
+    'label \N' \
+    'labelloc t c b' \
+    layer \
+    margin \
+    'nojustify true false' \
+    'ordering in out' \
+    'orientation 0.0' \
+    'penwidth 1.0' \
+    peripheries \
+    'pin false' \
+    pos \
+    rects \
+    'regular true false' \
+    'root false' \
+    'samplepoints 8' \
+    "shape $(string join ' ' -- (__fish_graphviz__print_shapes))" \
+    shapefile \
+    'showboxes 0 1 2' \
+    'sides 0' \
+    'skew 0' \
+    'sortv 0' \
+    'style filled invis' \
+    target \
+    tooltip \
+    URL \
+    vertices \
+    'width 0.75' \
+    xlabel \
+    xlp \
+    'z 0.0'
+
+complete -c $command -s G \
+    -a '(__fish_complete_key_value_pairs = $graph_attributes)' \
```

Comment:
I'm happy to hear other opinions, but my feeling is this is too clever by half. I don't think it's worth trying to provide this level of depth in completions.